### DISCUSSION TITLE
Support for PartNet Dataset

### DISCUSSION THREAD

**Discussion by beneisner** [pyg-team/pytorch_geometric#2791] [Ideas]
The [PartNet Dataset](https://partnet.cs.stanford.edu/) is a 3D dataset that provides a bunch of fine-grained semantic and instance labels for a large set of 3D objects. The dataset contains both meshes and point clouds, as well as hierarchical information about how the parts of objects are related. It is an extension of the ShapeNet effort, which this pytorch_geometric already supports.

I already have an implementation of this dataset that I'm using in my research, and would be happy to upstream it. Would such a contribution be welcome?

**Comment by rusty1s** [comment]
Definitely. Your contribution is very much welcome :)
